osu.Game/Screens/Play/HUD/MultiplayerGameplayLeaderboard.cs

```diff
@@ -65,7 +65,7 @@ private void load(OsuConfigManager config, IAPIProvider api)
                 var trackedUser = new TrackedUserData();
 
                 userScores[userId] = trackedUser;
-                var leaderboardScore = AddPlayer(resolvedUser, resolvedUser.Id == api.LocalUser.Value.Id);
+                var leaderboardScore = AddPlayer(resolvedUser, resolvedUser?.Id == api.LocalUser.Value.Id);
```

Comment:
Mind if I also annotate `AddPlayer`'s first param as `[CanBeNull]` for completeness?

Reply:
sure.